### PR TITLE
feat: [maps-mapsplatformdatasets] new module for maps-mapsplatformdatasets

### PR DESCRIPTION
**Pull request by emmileaf**
Initial generation for new mapsplatformdatasets client library: 

```
python3.9 generation/new_client/new-client.py generate \
  --api_shortname=maps-mapsplatformdatasets \
  --proto-path=google/maps/mapsplatformdatasets \
  --name-pretty="Maps Platform Datasets API" \
  --product-docs="https://developers.google.com/maps/documentation" \
  --api-description="Maps Platform Datasets API." \
  --cloud-api=false \
  --distribution-name="com.google.maps:google-maps-mapsplatformdatasets"
```

### CODE REVIEW

File: java-maps-mapsplatformdatasets/grpc-google-maps-mapsplatformdatasets-v1alpha/pom.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.api.grpc</groupId>
```
Comment (suztomo) [inline java-maps-mapsplatformdatasets/grpc-google-maps-mapsplatformdatasets-v1alpha/pom.xml:5]: This didn't became maps...

**Review by suztomo — CHANGES_REQUESTED** [review]
I may have to rewrite the generation script. No need to take action from you.

**Comment by suztomo** [comment]
@emmileaf Once https://github.com/googleapis/synthtool/pull/1770 is merged and you get necessary information via b/264293181, would you recreate the pull request?

**Comment by emmileaf** [comment]
> @emmileaf Once [googleapis/synthtool#1770](https://github.com/googleapis/synthtool/pull/1770) is merged and you get necessary information via b/264293181, would you recreate the pull request?

@suztomo Sounds good, thank you for the heads up!

**Pull request closed without merging**